decidim-templates/app/commands/decidim/templates/admin/destroy_questionnaire_template.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Templates
+    module Admin
+      class DestroyQuestionnaireTemplate < DestroyTemplate
+        protected
+
+        def destroy_template
+          Decidim.traceability.perform_action!(
+            :delete,
+            template,
+            current_user
+          ) do
+            template.destroy!
+            template.templatable.destroy
```

Comment:
Is this normal? should we destroy the templatable here for questionnaires?

Reply:
There was an after_destroy call on template model, that i have removed, and the line above is added to maintain the functionality.

Reply:
thanks, we will leave like this for now then. Although I suspect is not the best of ideas this behaviour